internal/filter/docker/docker.go

```diff
@@ -14,7 +14,7 @@ const Name = "docker"
 var Help = `
 docker:<image>
 
-Produce versions from a image on ducker hub or other registry.
```

Comment:
This was funny one.

Reply:
maybe should have kept it? 😄 🦆

Reply:
Ok, Google

<img width="424" alt="image" src="https://github.com/user-attachments/assets/8c6ac507-9ff2-4d01-a294-5d90fa2002c6">